pkg/kubelet/apis/config/helpers_test.go

```diff
@@ -135,6 +135,8 @@ var (
 		"TLSCertFile",
 		"TLSPrivateKeyFile",
 		"ResolverConfig",
+		"SeccompProfileRoot",
+		"VolumePluginDir",
```

Comment:
nit: the `VolumePluginDir` change could be handled as a separate PR.

Reply:
Agreed